RushHour/RushHourMod.cs

```diff
@@ -22,13 +22,17 @@ public void OnSettingsUI(UIHelperBase helper)
                 new OptionsCheckbox() { readableName = "EXPERIMENTAL: Use improved commercial demand", value = false, uniqueName = "UseImprovedCommercial" }
             };
 
+            loadSettingsFromSaveFile();
```

Comment:
Hmm in `LoadingExtension.cs` there is a line

```c#
CimToolsHandler.CimToolBase.XMLFileOptions.Load();
```
I feel like this should be the one doing this?